src/sentry/tasks/post_process.py

```diff
@@ -891,6 +892,12 @@ def _get_replay_id(event):
     if not replay_id:
         return
 
+    # Validate the UUID.
+    try:
+        uuid.UUID(replay_id)
+    except (ValueError, TypeError):
```

Comment:
should we increment a metric here?

Reply:
I was thinking about this earlier but couldn't come up with a use case for why I would want to know it.  Do you have anything in mind?

Reply:
how often this happens so we can prioritize a solution if needed.